guides/v2.3/comp-mgr/cli/cli-upgrade.md

```diff
@@ -10,128 +10,105 @@ functional_areas:
-Both ways require that you comply with the pre-upgrade checklist.
+Both scenarios require that you comply with the [Pre-upgrade checklist].
 
 {:.bs-callout .bs-callout-info}
-If you cloned the Magento 2 GitHub repository, you **cannot** use this method to upgrade. Instead, see [Update the Magento application].
-
-## Pre-upgrade checklist
-
-Comply with the the [Update and upgrade checklist].
-
-### Prerequisite for the `pub` directory root {#upgrade-cli-pub}
+If you cloned the Magento 2 GitHub repository, you **cannot** use this method to upgrade. Instead, see [Update the Magento application] for upgrade instructions.
 
-This section applies to you *only* if you set the Magento root directory to `<your Magento install dir>/pub`.
-If you did not do this, skip this section and continue with the next section.
+## Prerequisites
 
-For the upgrade, create another subdomain or docroot that uses the Magento installation directory as its root.
-Run the [System Upgrade utility] using that subdomain.
+Complete the following prerequisites to prepare your environment before starting the upgrade process:
+* **Complete Update and upgrade checklist**—To avoid possible errors during installation or upgrading, complete the [Update and upgrade checklist].
+* **Set the `pub` directory root**—If you set the Magento root directory to `<your Magento install dir>/pub`, create another subdomain or docroot that uses the Magento installation directory as its root, and run the [System Upgrade utility] using that subdomain.
+* **Check PHP and environment settings**—Verify that your PHP and other environment settings are compatible with the [system requirements].
+* **Switch to maintenance mode**—To prevent access to your store while it's being upgraded, switch your store to maintenance mode:
 
-### PHP and other environment settings
+    ```bash
+    php <your Magento install dir>/bin/magento maintenance:enable
+    ```
 
-Check PHP and other environment settings to be compatible with the [system requirements].
+    See [Enable or disable maintenance mode] for additional options.
 
-### Switch your store to the maintenance mode {#upgrade-cli-maint}
-
-To prevent access to your store while it's being upgraded, put your store in the maintenance mode.
-
-```bash
-php <your Magento install dir>/bin/magento maintenance:enable
-```
-
-For additional options, see [Enable or disable maintenance mode].
-
-{:.bs-callout .bs-callout-info}
-You can optionally create a [custom maintenance mode page].
+    {:.bs-callout .bs-callout-info}
+    Optionally, you can create a [custom maintenance mode page].
 
 ## Upgrade using the command line {#upgrade-cli-upgr}
+
+Using the more manual process of upgrading via the command line allows you to track and control exactly what's being changed in the upgrade. If you previously made updates to the same values that the upgrade script affects, the script will override those values, so using the manual process is the best approach.
```

Comment:
Removed my previous comment after reading all sections. Updated feedback:

Maybe add a tip here pointing to the script process as easier and preferred if you do not have modified values? You have a nice callout in the script section highlighting this process in case of modified values, something along those lines here may be helpful?